### DISCUSSION TITLE
Number of Proteins.Identified Vs proteins in report.pg_matrix

### DISCUSSION THREAD

**Discussion by witszymanski** [vdemichev/DiaNN#243] [Q&A]
Hi all, 
I have a small question because I can't really find the answer on the github page. In the `report.stats.tsv` I can find the numbers of identified proteins and all my samples have around 2100. 
![image](https://user-images.githubusercontent.com/15866068/141776648-2fc32072-30a1-4ea7-ba17-29bfc3a909a2.png)

However, when I look at the `report.pg_matrix.tsv` table, I can find around 4000 rows and the distribution among samples looks like this: 
![image](https://user-images.githubusercontent.com/15866068/141776841-81d49f0f-1dd5-42df-8fa0-18ad0dbb6304.png)
(sample names are different because I renamed them in R) 
Can you explain me these numbers? 

Side question: is this the number of commonly used "points per peak" ?
![image](https://user-images.githubusercontent.com/15866068/141777004-974ce90d-e9af-4198-abf3-b0122a774294.png)

All the best
Witek

**Comment by vdemichev** [comment]
Hi Witek, 

- Filtering is different, also in the .stats file it's uniquely identified proteins that are counted.
- No, it's peak width as measured in SWATH cycles. The average number of data points per peak is printed by DIA-NN in the log when it's run with scan window = 0.

Best,
Vadim

**Reply by vdemichev** [reply]
I updated the docs with the description of how the number of proteins in the stats file is calculated

**Reply by witszymanski** [reply]
> Filtering is different, also in the .stats file it's uniquely identified proteins that are counted.

Ok, Thanks! I'll check the new description! 

> No, it's peak width as measured in SWATH cycles. The average number of data points per peak is printed by DIA-NN in the log when it's run with scan window = 0.

Ok, I have the log files and it was ran with `scan window = 0` . Where do I find it now? How is the line called?

**Reply by vdemichev** [reply]
'Peak width'
